src/main/scala/diplomaticobjectmodel/DiplomaticObjectModelUtils.scala

```diff
@@ -6,6 +6,7 @@ import java.io.{File, FileWriter}
 
 import Chisel.{Data, Vec, log2Ceil}
 import freechips.rocketchip.diplomacy.{AddressRange, AddressSet, Binding, Device, DiplomacyUtils, ResourceAddress, ResourceBindings, ResourceBindingsMap, ResourceInt, ResourceMapping, ResourcePermissions, ResourceValue, SimpleDevice}
+import freechips.rocketchip.diplomaticobjectmodel._
```

Comment:
This wildcard import presumably is unnecessary because nothing else in this file changed.

Reply:
Fixed